app/javascript/components/data-tables/requests-table/index.jsx

```diff
@@ -0,0 +1,140 @@
+/* eslint-disable no-undef */
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import MiqDataTable from '../../miq-data-table';
+
+/* const TimeSinceCreation = (currentTime, creationTime) => {
+  const elapsed = currentTime - new Date(creationTime).getTime();
+
+  if (Math.floor(elapsed / 1000 / 60000000) >= 1) {
+    return "test"
+  }
+  else if (Math.floor(elapsed / 1000 / 60) >= 1) { return __(`${Math.floor(elapsed / 1000 / 60)} minutes ago`) }
+  else if (Math.floor(elapsed / 1000 / 60) === 1) { return __(`${Math.floor(elapsed / 1000 / 60)} minute ago`) }
+  else if (Math.floor(elapsed / 1000) > 1) { return __(`${Math.floor(elapsed / 1000)} seconds ago`) }
+  else if (Math.floor(elapsed / 1000) === 1) { return __(`${Math.floor(elapsed / 1000)} second ago`) }
+  else { return __(`${elapsed} miliseconds ago`)}
+
+} */
```

Comment:
@DavidResende0 Can this be removed?

Reply:
Right now the logs display the actual timestamp in the table. This function's purpose was to convert that into a x seconds/minutes/hours ago statement. It technically works, but it's not perfect and I found that it didn't quite work great in situations where all the logs basically happened within a very short time of each other (< 60 sec). We can remove it for now and discuss adding it in the future if you want

Reply:
Yeah, I'd rather we didn't commit commented code and we can always open a follow up PR with it.